neqo-transport/src/connection/params.rs

```diff
@@ -209,6 +211,15 @@ impl ConnectionParameters {
         self.idle_timeout
     }
 
+    pub fn get_max_datagram_frame_size(&self) -> u64 {
+        self.max_datagram_frame_size
+    }
+
+    pub fn max_datagram_frame_size(mut self, v: u64) -> Self {
```

Comment:
```suggestion
    pub fn datagram_size(mut self, v: u64) -> Self {
```

Shorter names = good.